src/main.py

```diff
@@ -300,6 +299,7 @@ def run(self):
                                     sub.sticky()
                                     print "Submission stickied..."
                             break
+                        else: time.sleep(120)

```

Comment:
This change does not increase the rate of requests to the Reddit API with every loop. The str variable (line 249) holds the text used to edit the game thread, and all this change does is move the 120 second wait until after str is checked for |Decisions|. This change will result in hits to the Reddit API more often than 120 seconds only once per game, when the game is over.

Decreasing the sleep between game thread edits would be welcome, but I think that could lead to rate limit issues and I think it should be separate from this change.

I've been running the r/Phillies bot with this change for a year and I've never hit the rate limit.